app/src/main/java/com/example/roomremix2/account.kt

```diff
@@ -16,6 +16,4 @@ class account : AppCompatActivity() {
 
         binding.
     }
-
-
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.